tools/accuracy_checker/accuracy_checker/annotation_converters/fewrel.py

```diff
@@ -0,0 +1,389 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from collections import namedtuple
+
+import numpy as np
+
+from ..representation import TextClassificationAnnotation, RelationClassificationAnnotation
+from ..utils import read_json
+from ..config import PathField, NumberField, BoolField
+
+from .format_converter import BaseFormatConverter, ConverterReturn
+from ._nlp_common import get_tokenizer, CLS_ID, SEP_ID
+
+
+class ERNIEFewrelConverter(BaseFormatConverter):
+    __provider__ = "ernie_fewrel"
```

Comment:
ernie is the model name or dataset?

Reply:
FewRel is name of dataset. ERNIE is a BERT-like model. I called converter "ernie_fewrel" because its output suitable only for combination of FewRel dataset and ERNIE model. Converter uses "hardcoded" data from second part of model, KG (knowledge graph).

Reply:
ok